tools/README.md

```diff
@@ -0,0 +1,17 @@
+# TaskTupleAwaiter tools
+
+these folders contain tools for the project
+
+Generator.CLI can be used like 
+
+ - ```generator d D:\ -v false -a 20 --format true```
+ - ```generator f D:\FileName.cs -v false -a 20 --format true```
```

Comment:
Not important, but you only need more than one backtick for inline code if the code itself contains a backtick.